README.md

```diff
@@ -28,13 +28,15 @@ Add the following to your `.eslintrc` config:
       // turn on errors for missing imports
       "import/no-unresolved": "error"
     },
-    "import/resolver": {
-      // use <root>/tsconfig.json
-      "typescript": {},
-
-      // use <root>/path/to/folder/tsconfig.json
-      "typescript": {
-        "directory": "./path/to/folder"
+    "settings": {
```

Comment:
As far as I know, yes. Eslint throws an error otherwise as @levithomason mentioned. 

FWIW: your repositories are configured this way: [alexgorbatchev/typescript-module-boilerplate/.eslintrc.json#L17-L25](https://github.com/alexgorbatchev/typescript-module-boilerplate/blob/3d02a699491c80ef3033b668ec520e82aad4f0e1/.eslintrc.json#L17-L25)

Reply:
Maybe the top level `settings` is not necessary then? It feels odd that there's settings node within settings node.

Reply:
I understand what you are saying now. I didn't even see the `settings` key at the very top. I pushed the necessary changes.